packages/@react-aria/utils/src/index.ts

```diff
@@ -35,3 +35,5 @@ export {scrollIntoView, scrollIntoViewport} from './scrollIntoView';
 export {clamp, snapValueToStep} from '@react-stately/utils';
 export {isVirtualClick, isVirtualPointerEvent} from './isVirtualEvent';
 export {useEffectEvent} from './useEffectEvent';
+export {useIsMobileDevice} from './useIsMobileDevice';
+export {useHasChild} from './useHasChild';
```

Comment:
Now that you have `useIsMobileDevice` and `useHasChild` in the `aria/utils` package, you can delete the `spectrum/utils` versions and re-export them here https://github.com/adobe/react-spectrum/blob/bb5259d1bdcd0ce5e9d3636af64e56733a6e5a04/packages/%40react-spectrum/utils/src/index.ts#L36